docusaurus/docusaurus.config.js

```diff
@@ -13,8 +13,8 @@ const config = {
     // Assumed relative path.  If you are using airbytehq.github.io use /
     // anything else should match the repo name
     baseUrl: '/',
-    onBrokenLinks: 'warn',
-    onBrokenMarkdownLinks: 'warn',
+    onBrokenLinks: 'throw',
```

Comment:
these lines make it so we can't deploy with broken links unless you specify a command to ignore the warnings